src/Expensify.js

```diff
@@ -6,7 +6,7 @@ import {AppState, Linking} from 'react-native';
 import Onyx, {withOnyx} from 'react-native-onyx';
 import * as Report from './libs/actions/Report';
 import BootSplash from './libs/BootSplash';
-import * as ActiveClientManager from './libs/ActiveClientManager';
+import ActiveClientManager from './libs/ActiveClientManager';
```

Comment:
I would prefer to leave the imports the old way and keep the PR more focused to just convert the files to TS.